pom.xml

```diff
@@ -4,9 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <groupId>com.autos</groupId>
     <artifactId>auto-project</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0</version>
```

Comment:
can you please explain, why do we need to remove snapshot word?

Reply:
because this version is not  snapshot

Reply:
thanks, i've understood